Migrations/20210407094917_created-at-column-add.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace whale_spotting.Migrations
+{
+    public partial class createdatcolumnadd : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
+                name: "CreatedAt",
+                table: "Sightings",
+                type: "timestamp without time zone",
+                nullable: false,
+                defaultValue: new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified));
```

Comment:
I think we would want this to have a default value of the current time instead of just the start of time - just as otherwise we might have surprising data for all the data, ideally it would actually be defaulted to Sighted_at if you can figure out how to do that